.travis.yml

```diff
@@ -39,3 +39,5 @@ env:
 notifications:
   slack:
     secure: H5nS+GX6TYTU27ur6YFG5OgrQeUbzXLok5ub6+xcmyYEeVPpnQ1Gg/wKqTAGsP9j6tAkqPpxgYT9i9Do6eyTEplK6bTvQVyhilsEDtxGJbUO8XOE9TSo6jAe/lD3EB5l46gxFID+Hg9IkPii4LwEabP7PVehrB1JfNZ6QDgSRRM=
+after_failure:
+  - cat ./genie-client/build/test-results/test/TEST-com.netflix.genie.client.ClusterClientIntegrationTests.xml
```

Comment:
Is this temporary? If not, should print out all the output files under test-results/test.